pandas/tests/arrays/boolean/test_arithmetic.py

```diff
@@ -46,8 +47,11 @@ def test_add_mul(left_array, right_array, opname, exp):
 
 
 def test_sub(left_array, right_array):
-    with pytest.raises(TypeError):
-        # numpy points to ^ operator or logical_xor function instead
+    msg = re.escape(
```

Comment:
ok great. yeah these are expected. i don't like the external_ thing because it is not obvious what is happening.

Reply:
I don't like it much either but it's what the guidance in the issue says to do. *shrug*

Reply:
hah, that's why I look at all the PRs :->